exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,37 @@
+var bracket = function() {
```

Comment:
A couple suggestions that were passed on to me: 

1) Try to avoid using 'var', and instead use 'let'
     -I don't remember the why, but JRob told me, so there must be good reasons
2) Avoid using '==', although at times you might have to, and instead use '==='